test/fgpu/test_engine.py

```diff
@@ -965,7 +964,7 @@ async def test_steady_state_gain(
         monkeypatch,
     ) -> None:
         """Test that the ``steady-state-timestamp`` is updated correctly after ``?gain``."""
-        n_samples = max(8 * CHUNK_SAMPLES, output.spectra_samples * SPECTRA_PER_HEAP * 3)
+        n_samples = max(16 * CHUNK_SAMPLES, output.spectra_samples * SPECTRA_PER_HEAP * 3)
```

Comment:
Not important in the grand scheme of updates/changes, but even in this silo'd context of reading the commit message as
> The output chunks were too large, causing the gains to be applied to the first chunk.

Here it seems like you're ensuring more data is pushed through the digitiser stream. How does this address the reasoning for the commit?
* My initial impression/thought was expecting a resizing/shrinking of Output Chunk size.